tests/conftest.py

```diff
@@ -155,3 +155,28 @@ def pytest_configure(config):
             enabled_remotes.discard(remote_name)
         if enabled:
             enabled_remotes.add(remote_name)
+
+
+@pytest.fixture
+def git_upstream(tmp_dir, erepo_dir, git_dir, request):
```

Comment:
It's probably better to put these in `tests/dir_helpers.py` with the other `tmp_dir` related fixtures

Reply:
I'm just going to push this change.